test/runtime/connectivity.go

```diff
@@ -320,9 +320,13 @@ var _ = Describe("RuntimeValidatedConntrackTest", func() {
 		Expect(err).Should(BeNil(), fmt.Sprintf("could not get metadata for container %q", curl2ContainerName))
 		By(fmt.Sprintf("httpd1 container Docker networking: %s", curl2DockerNetworking))
 
+		By("Showing policies imported to Cilium")
+		res := vm.PolicyGetAll()
+		fmt.Printf("%s\n", res.CombineOutput())
```

Comment:
Just to be sure, is this a leftover?

Reply:
I left this because it clearly displays what policies are imported in Cilium to console without wrapping it in anything else - I can remove it if you like, though.